MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Calendar.xaml

```diff
@@ -314,7 +314,7 @@
                         </DataTemplate>
                     </ControlTemplate.Resources>
 
-                    <Grid x:Name="PART_Root">
+                    <Grid x:Name="PART_Root" Cursor="">
```

Comment:
Do you mean to set the Cursor property to empty string here?

Reply:
tbh no, I think it did that by itself or something...